project-base/app/config/packages/overblog_graphql.yaml

```diff
@@ -0,0 +1,6 @@
+overblog_graphql:
```

Comment:
this configuration key is already in the file `project-base/app/config/packages/shopsys_frontend_api.yaml`
It would be nice to unify it

Reply:
I have split shopsys_frontend_api.yaml to separate files so it is more consistent